scripts/run_clang_tidy

```diff
@@ -8,6 +8,7 @@ bazel_build_flags+=(
   '--experimental_action_listener=@io_kythe//kythe/cxx/tools/generate_compile_commands:extract_json'
   '--noshow_progress'
   '--noshow_loading_progress'
+  '--experimental_extra_action_top_level_only'
```

Comment:
What does it do?

Reply:
clang_tidy was not working, not really related to the PR. I am still not 100% sure why, but the "hack-ish" experimental code was trying to compile all bazel targets and getting into some GO code that it didn't configure well. This keeps it to only the first level of targets, which is exactly what we need.